composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "license": "GPL-3.0+",
     "require": {
-        "php": ">=5.4.0",
-        "ext-bcmath": "*"
+        "php": ">=5.4.0"
     },
     "require-dev": {
+        "ext-bcmath":                               "*",
```

Comment:
It's a little bit painful to use an alternative of a built-in php function.

Btw, I found this: http://php.net/manual/en/bc.requirements.php

> Since PHP 4.0.4, libbcmath is bundled with PHP. You don't need any external libraries for this extension.

This mean bcmath extension is generaly already enabled on php.

So if you don't want to remove the requirements, we can just close this PR and let is "as is". :+1: